projects/engine/src/connector.ts

```diff
@@ -1,25 +1,29 @@
 import { ExternalProfile, Profile } from '../../utils/src/types/profile'
 import { Message } from '../../utils/src/types/message'
-import { Plugin } from './plugin/abstract'
+import { Plugin, PluginOptions } from './plugin/abstract'
 
 /** List of available gateways for decentralised storage */
 export const defaultGateways = {
   'ipfs://': (url, name) => `https://${name}.dyn.plugin.remixproject.org/ipfs/${url.replace('ipfs://', '')}`,
-  'swarm://': (url, _) => `https://swarm-gateways.net/bzz-raw://${url.replace('swarm://', '')}`
+  'swarm://': (url, name) => `https://swarm-gateways.net/bzz-raw://${url.replace('swarm://', '')}`
 }
 
 /** Transform the URL to use a gateway if decentralised storage is specified */
-export function transformUrl(url: string, name: string) {
+export function transformUrl({ url, name }: Profile & ExternalProfile) {
```

Comment:
I "think" this is part of a previous PR.

Reply:
It is, but the "connector" file doesn't exist on master